app/SchemaValidator.php

```diff
@@ -16,8 +16,9 @@
  * @method static array|object getSchemaContents(string $relativeUri, bool $associative = true)
  * @method static void putSchemaContents(string $relativeUri, array|object $schema)
  * @method static string registerRawSchema(bool|object|string $schema)
- * @method static bool validate(array|object $data, string $schema)
- * @method static bool validateOrThrow(string|array|object $data, string $schema, string $exceptionMessage = null, bool $appendValidationDescriptions = false, int $failureHttpStatusCode = Response::HTTP_BAD_REQUEST)
+ * @method static bool validate(mixed $data, string $schema)
```

Comment:
The actual code in the Service for both these methods can also validate bools and numbers, but the type hint **here** was wrong.